### PR TITLE
Anydo.Anydo version 4.2.155

### PR DESCRIPTION
**Pull request by AmoghSingh25**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16563)

### CODE REVIEW

File: manifests/a/Anydo/Anydo/4.2.155.9593/Anydo.Anydo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Anydo.Anydo
+PackageVersion: 4.2.155.9593
```
Comment (ItzLevvie) [inline manifests/a/Anydo/Anydo/4.2.155.9593/Anydo.Anydo.installer.yaml:4]:
```suggestion
PackageVersion: 4.2.155
```
image

File: manifests/a/Anydo/Anydo/4.2.155.9593/Anydo.Anydo.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Anydo.Anydo
+PackageVersion: 4.2.155.9593
```
Comment (ItzLevvie) [inline manifests/a/Anydo/Anydo/4.2.155.9593/Anydo.Anydo.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 4.2.155
```
image

File: manifests/a/Anydo/Anydo/4.2.155.9593/Anydo.Anydo.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Anydo.Anydo
+PackageVersion: 4.2.155.9593
```
Comment (ItzLevvie) [inline manifests/a/Anydo/Anydo/4.2.155.9593/Anydo.Anydo.yaml:4]:
```suggestion
PackageVersion: 4.2.155
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the folder version using the GitHub Desktop client on Windows.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**